commit 03283b0c72cd3039a3e3d9d452256eab129e93c2
Author: John Spray <jspray@redhat.com>
Date:   Fri Mar 2 12:50:30 2018 -0500

    mgr: make PyModuleRegistry::init return void
    
    This was returning an int that was only ever zero.
    
    Signed-off-by: John Spray <john.spray@redhat.com>

diff --git a/src/mgr/PyModuleRegistry.cc b/src/mgr/PyModuleRegistry.cc
index 7285d02d7d3..8cf9162a4be 100644
--- a/src/mgr/PyModuleRegistry.cc
+++ b/src/mgr/PyModuleRegistry.cc
@@ -37,7 +37,7 @@ std::string PyModuleRegistry::config_prefix;
 
 
 
-int PyModuleRegistry::init()
+void PyModuleRegistry::init()
 {
   Mutex::Locker locker(lock);
 
@@ -98,8 +98,6 @@ int PyModuleRegistry::init()
     clog->error() << "Failed to load ceph-mgr modules: " << joinify(
         failed_modules.begin(), failed_modules.end(), std::string(", "));
   }
-
-  return 0;
 }
 
 bool PyModuleRegistry::handle_mgr_map(const MgrMap &mgr_map_)
diff --git a/src/mgr/PyModuleRegistry.h b/src/mgr/PyModuleRegistry.h
index 16ec132bfbd..e2221affe47 100644
--- a/src/mgr/PyModuleRegistry.h
+++ b/src/mgr/PyModuleRegistry.h
@@ -86,7 +86,7 @@ public:
    */
   bool handle_mgr_map(const MgrMap &mgr_map_);
 
-  int init();
+  void init();
 
   void active_start(
                 PyModuleConfig &config_,
